src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -460,12 +460,12 @@ public void delete(DeleteRequest request, ActionListener<DeleteResponse> listene
          * @see Requests#searchRequest(String...)
          */
         public void search(SearchRequest request, ActionListener<SearchResponse> listener) {
-            restHighLevelClient.searchAsync(request, RequestOptions.DEFAULT, listener);
+            RequestOptions options = RequestOptions.DEFAULT.toBuilder().addHeader("Authorization", "Bearer TOKEN_HERE").build();
```

Comment:
Thanks for showing the code on how to do this.  Can we make these two Strings something we set on the client?